### DISCUSSION TITLE
how to use fast-dds-monitor

### DISCUSSION THREAD

**Discussion by YGD159** [eProsima/Fast-DDS-monitor#150] [Q&A] [answered]
How to deal with the problem?
I can't find the infomation about physical.
![lALPDhmOwlTjAJzNA6fNBak_1449_935 png_620x10000q90g](https://user-images.githubusercontent.com/52145923/130193802-1422adc1-0def-4e31-9679-4d204508d58d.jpg)

**Comment by jparisu** [comment, marked as answer]
Hi @YGD159 

Glad to know that you are using Fast DDS Monitor.

My first approach is that you have not enabled statistics when configuring the Shapes Demo application.
Be sure that you active the checkbox `Active Statistics` in the `Options->Participant Configuration` window: https://eprosima-shapes-demo.readthedocs.io/en/stable/first_steps/first_steps.html#participant-configuration .
In case the Participant is already running, you would need to `Stop` and `Start` the Participant in order for this configuration to take place.

In case this does not fix the issue, please retrieve further information about the configuration that you are using in both Shapes Demo and Fast DDS Monitor, and the steps that you have followed.
